hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneFSUtils.java

```diff
@@ -210,7 +210,7 @@ public static int getFileCount(String keyName) {
 
 
   public static boolean isFSOptimizedBucket(BucketLayout bucketLayout) {
```

Comment:
Thank you @mukul1987 for the patch. Nice refactoring!

Can you please remove the OzoneFSUtils#isFSOptimizedBucket() method and all the callers can directly use `bucketLayout.isFileSystemOptimized()`.

OzoneFSUtils#isFSOptimizedBucket() method used in following functions.
```
BasicRootedOzoneFileSystem.rename(Path, Path) 
TrashOzoneFileSystem.rename(Path, Path)  
BasicRootedOzoneFileSystem.delete(Path, boolean) 
TrashOzoneFileSystem.delete(Path, boolean) 
```

Reply:
done